lab_1/main.py

```diff
@@ -51,6 +112,35 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     """
     pass
 
+    if not isinstance(freq_dict, dict) or not isinstance(top_n, int):
+        return []
+
+    # the number of words (from the most popular)
+    values_of_words = []
+    for x in freq_dict.values():
+        if x in values_of_words:
+            continue
+        values_of_words.append(x)
+    values_of_words.sort()
+    values_of_words = values_of_words[::-1]
+    # top_values
+    top = values_of_words[:top_n]
+    #
+    top_words = []  # can be words with the same frequency
+    for i in top:
+        for word, frequency in freq_dict.items():
+            if i == frequency:
+                top_words.append(word)
+    top_words = top_words[: top_n]  # unique
+    return top_words
+
+
+top_words = get_top_n_words(freq_dict, top_n)
```

Comment:
There should be no function calling\using in `main.py` -> use functions on the`start.py`.